sequence.go

```diff
@@ -26,6 +28,19 @@ func (s *Sequence) Value(state ReadOnlyState) uint64 {
 	return seq
 }
 
+func (s *Sequence) Value2(kvStore store.KVReader) uint64 {
```

Comment:
What's the point of this function, the `ReadOnlyState` already implements the `KVReader` interface.

Reply:
Right remove `Value2`